### PR TITLE
[ci][webui] Enable Style/ZeroLengthPredicate rubocop cop

### PR DESCRIPTION
**Pull request by DavidKang**
Checks for receiver.length == 0 predicates and the negated versions receiver.length > 0 and receiver.length != 0.

### CODE REVIEW

File: src/api/app/controllers/webui/request_controller.rb [hunk 1]
```diff
@@ -100,7 +100,7 @@ def show
     @my_open_reviews = @req['my_open_reviews']
     @other_open_reviews = @req['other_open_reviews']
     # rubocop:disable Metrics/LineLength
-    @can_add_reviews = %w(new review).include?(@state) && (@is_author || @is_target_maintainer || @my_open_reviews.length > 0) && !User.current.is_nobody?
+    @can_add_reviews = %w(new review).include?(@state) && (@is_author || @is_target_maintainer || !@my_open_reviews.empty?) && !User.current.is_nobody?
```
Comment (Ana06) [inline src/api/app/controllers/webui/request_controller.rb]: What about `.present` instead `! .empty`?

File: src/api/app/models/owner.rb [hunk 2]
```diff
@@ -355,7 +355,7 @@ def self._extract_from_container(m, r, sql, objfilter)
   def self._build_rolefilter_sql(rolefilter)
     # construct where condition
     sql = nil
-    if rolefilter.length > 0
+    if !rolefilter.empty?
```
Comment (Ana06) [inline src/api/app/models/owner.rb]: What about `.present` instead `! .empty`?

File: src/api/app/models/channel.rb [hunk 3]
```diff
@@ -150,7 +150,7 @@ def is_active?
     # no targets defined, the project has some
     return true if channel_targets.size.zero?
 
-    channel_targets.where(disabled: false).size > 0
+    !channel_targets.where(disabled: false).empty?
```
Comment (Ana06) [inline src/api/app/models/channel.rb]: What about `.present` instead `! .empty`?

File: src/api/app/models/repository.rb [hunk 4]
```diff
@@ -73,7 +73,7 @@ def cleanup_before_destroy
     linking_repositories.each do |lrep|
       lrep.path_elements.includes(:link, :repository).each do |pe|
         next unless pe.link == self # this is not pointing to our repo
-        if lrep.path_elements.where(repository_id: Repository.deleted_instance).size > 0
+        if !lrep.path_elements.where(repository_id: Repository.deleted_instance).empty?
```
Comment (Ana06) [inline src/api/app/models/repository.rb]: What about `.present` instead `! .empty`?

File: src/api/app/models/repository.rb [hunk 5]
```diff
@@ -89,7 +89,7 @@ def cleanup_before_destroy
       lrep.targetlinks.includes(:target_repository, :repository).each do |rt|
         next unless rt.target_repository == self # this is not pointing to our repo
         repo = rt.repository
-        if lrep.targetlinks.where(repository_id: Repository.deleted_instance).size > 0
+        if !lrep.targetlinks.where(repository_id: Repository.deleted_instance).empty?
```
Comment (Ana06) [inline src/api/app/models/repository.rb]: What about `.present` instead `! .empty`?

File: src/api/app/models/status_history_rescaler.rb [hunk 6]
```diff
@@ -32,7 +32,7 @@ def find_items_for_maxtime(key, maxtimeoffset)
   def find_start_items(allitems, max)
     items = []
 
-    items << allitems.shift while allitems.length > 0 && allitems[0].time < max
+    items << allitems.shift while !allitems.empty? && allitems[0].time < max
```
Comment (Ana06) [inline src/api/app/models/status_history_rescaler.rb]: What about `.present` instead `! .empty`?

File: src/api/app/policies/group_policy.rb [hunk 7]
```diff
@@ -13,7 +13,7 @@ def update?
     return true if @user.is_admin?
 
     # and also group maintainers
-    @record.group_maintainers.where(user: @user).length > 0
+    !@record.group_maintainers.where(user: @user).empty?
```
Comment (Ana06) [inline src/api/app/policies/group_policy.rb]: What about `.present` instead `! .empty`?

File: src/api/.rubocop.yml [hunk 8]
```diff
@@ -141,6 +141,9 @@ Style/SpaceInsideBrackets:
 Style/TernaryParentheses:
   Enabled: true
 
+# Checks for receiver.length == 0 predicates and the negated versions receiver.length > 0 and receiver.length != 0.
```
Comment (Ana06) [inline src/api/.rubocop.yml]: I know that this is what the Rubocop documentation says, but it is not accurate as it also checks for things like `.length < 1` and it also works for `size`.

File: src/api/app/controllers/webui/request_controller.rb (hunk 1 above)
Comment (DavidKang) [inline src/api/app/controllers/webui/request_controller.rb]: 🤔 .... yes, why not. I will change it in another PR

File: src/api/.rubocop.yml (hunk 8 above)
Comment (DavidKang) [inline src/api/.rubocop.yml]: right!

Comment (Ana06) [inline src/api/.rubocop.yml]: what about _Checks for predicates than can be replaced by receiver.empty? and !receiver.empty?_ ?

**Pull request merged**